cmd/swarm/config.go

```diff
@@ -304,6 +308,12 @@ func envVarsOverride(currentConfig *bzzapi.Config) (config *bzzapi.Config) {
 		}
 	}
 
+	if max := os.Getenv(SWARM_ENV_MAX_STREAM_PEER_SERVERS); max != "" {
+		if m, err := strconv.Atoi(max); err == nil {
```

Comment:
should we not fatal if `Atoi` fails?

Reply:
Function envVarsOverride is not returning errors, but we can os.Exit on error in it, as it is a cmd package. Now I see that in this function on other places, we are setting values only if there is error, which is not right:

```
		if swap, err := strconv.ParseBool(swapenable); err != nil {
			currentConfig.SwapEnabled = swap
		}
```

Should we os.Exit on all parsing errors in this function instead?

Reply:
ooops

Reply:
I have updated the handling for this particular environment variable, and will create an issue for other cases, to be solved in another pr.